tools/unitctl/README.md

```diff
@@ -51,11 +51,44 @@ desired.
 
 ## Features (Current)
 
+```
+CLI interface to the NGINX UNIT Control API
+
+Usage: unitctl [OPTIONS] <COMMAND>
+
+Commands:
+  instances  List all running UNIT processes
+  edit       Open current UNIT configuration in editor
+  import     Import configuration from a directory
+  execute    Sends raw JSON payload to UNIT
+  status     Get the current status of UNIT
+  listeners  List active listeners
+  help       Print this message or the help of the given subcommand(s)
+
+Options:
+  -s, --control-socket-address <CONTROL_SOCKET_ADDRESS>
+          Path (unix:/var/run/unit/control.sock), tcp address with port (127.0.0.1:80), or URL
```

Comment:
Do the tcp address and URLs require a scheme identifier? http:// or https://? Or if there's no scheme (ie it doesn't start with `unix:`, the ctl will assume it's one of the two?

Reply:
the TCP address does not require a scheme.

The logic is in unit-client-rs/src/control_socket_address.rs (pub fn parse_address).
It looks like if the string does not start with unix:, unix://, or @ then it will attempt to parse a TCP socket.